image-based-installation-config-template.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Note: This is a sample ImageBasedInstallConfig file showing
```

Comment:
We should also rename the template used for DHCP :)

Reply:
I've renamed the `image-based-config-template-dhcp.yaml` [here](https://github.com/rh-ecosystem-edge/ib-orchestrate-vm/pull/82/files#diff-4efd84bdfa97dd88e4c7b6b6995ccd7e365fd6d137291df157c081dc661d68d6). 

Do we also need an `image-based-installation-config-template-dhcp.yaml`?